cmd/server/cmd/server.go

```diff
@@ -218,6 +227,27 @@ func runServer(sa *serverArgs, repo template.Repository, adapters []pkgAdapter.I
 		}
 		eval = ilEval
 	}
+
+	var dispatcher mixerRuntime.Dispatcher
+
+	// TODO until the dispatcher 2 switch is complete,
+	// dispatcher 2 is only enabled when configStore2URL is specified.
+	if sa.configStore2URL != "" {
+		adapterMap := adapter.InventoryMap(adapters)
+		store2, err := store.NewRegistry2(config.Store2Inventory()...).NewStore2(sa.configStore2URL)
+		if err != nil {
+			fatalf("Failed to connect to the configuration2 server. %v", err)
+		}
+		dispatcher, err = mixerRuntime.New(eval, gp, adapterGP,
```

Comment:
should we just call this runtime ?

Reply:
When you are reading code, `runtime` can be confusing, it feel like the builtin runtime.